xcode/Subconscious/Shared/Parsers/Subtext.swift

```diff
@@ -640,17 +640,15 @@ extension Subtext {
 }
 
 extension Subtext {
-    private static let maxExcerptSize = 280
-    private static let maxExcerptBlocks = 2
+    private static let maxExcerptSize = 1024
```

Comment:
Simplify the approach, we take more text than we need for any view regardless of the number of blocks.